includes/utilities.inc

```diff
@@ -460,11 +460,13 @@ function islandora_paged_content_ocr_combine(array $files, $out) {
   $success = TRUE;
   foreach ($files as $file) {
     $ocr = file_get_contents($file);
-    if ($ocr == FALSE) {
+    if ($ocr === FALSE) {
       $success = FALSE;
     }
     else {
-      $success = file_put_contents($out, $ocr, FILE_APPEND) == FALSE ? FALSE : $success;
+      if ($ocr) {
```

Comment:
`$ocr` would indeed `=== "0"`, but `if ($ocr)` isn't a strict comparison; it's the same as asking `if ($ocr == TRUE)` which doesn't match the string '0'.

I tested this by generating some files and seeing how they concatenated together:

```php
// Our test items.
$files = array(
  '/tmp/a.txt' => 'A',
  '/tmp/b.txt' => 'B',
  '/tmp/c.txt' => '0',
  '/tmp/d.txt' => 'D',
);
$out = '/tmp/out.txt';
// Write all of these.
foreach ($files as $path => $content) {
  file_put_contents($path, $content);
}
// Give us an empty output file.
file_put_contents($out, '');

// Attempt to write our files to $out.
module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
islandora_paged_content_ocr_combine(array_keys($files), $out);
// This yields "ABD"; 0 gets missed.
print_r(file_get_contents($out));
```

Modifying to the following yields "AB0D" instead to `$out`:

```php
[...]
if ($ocr === FALSE) {
  $success = FALSE;
}
else {
  $success = file_put_contents($out, $ocr, FILE_APPEND) === FALSE ? FALSE : $success;
}
[...]
```